README.md

```diff
@@ -1,14 +1,37 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S2-CS2103T-W14-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103T-W14-3/tp/actions)
+
+
+
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info. 
+# InternBuddy
+InternBuddy is a desktop app for managing companies for internships, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+If you can type fast, InternBuddy can get your internship management deliverables done faster than traditional GUI apps.
+
+InternBuddy provides you a way to plan and organize all your internship details on a single application, with intuitive 
+UI to make your Job Seeking journey a smooth road.
+* Store and manage Companies
+* Manage Contacts 
+  * For example:  Hiring Manager Contact Details
+* Manage Events
+  * For example: Interviews and Online Assessments
+
+For more detailed information on how to use the application, click on [User Guide](docs/UserGuide.md)
+
+---
+
+### More Information about the Project
+* InternBuddy is an **ongoing** collaborated Software Development Project maintained by 
+  * Albert Sutiono
+  * Edward Alvin
+  * Muhammad Radhya
+  * Steven Gabriel Kho Chua
+* InterBuddy is a brown-field project which is forked from [Address Book 3](https://github.com/nus-cs2103-AY2122S2/tp), a software development project (around 6 KLoC) which provide a baseline for InternBuddy.
```

Comment:
Minor typo here, think it should be InternBuddy instead of InterBuddy

Reply:
Fixed